src/utils/formHelpers.js

```diff
@@ -0,0 +1,18 @@
+import Joi from "joi";
+
+export function getErrorClass(error) {
+  if (!error) {
+    return null;
```

Comment:
you can return '' or undefined here and it wont change the screenshots. undefined preferred but '' if you have to, like i mentioned before we prefer null/undefined for not present unless we rly cant use it.

Reply:
I remember that you said before to change it to `null` and not a good practice to return an empty string, that make sense really. I change the function call to:
```
getErrorClass(error)  || ''
```

Reply:
right but you dont want to change that function all everywhere either so i was partially mistaken in this scenario. that also counts as copy-pasting code if you do that everywhere.

returning undefined should give what you want though.